src/parser/WDL/parse.js

```diff
@@ -66,11 +436,22 @@ export default function parse(data) {
   };
 
   let ast;
-  if (result.status) {
-    const ret = hermesStage(data);
-    result.status = ret.status;
-    result.message = ret.message;
-    ast = ret.ast;
+  const ret = hermesStage(data);
+  result.status = ret.status;
+  result.message = ret.message;
```

Comment:
We should consider revising the approach to returning results, as we change `parse()` method to an async promise-based implementation. Errors should be thrown to be caught in a `.catch()` clause.

Reply:
Fixed.